### PR TITLE
fix responsive on load issue.

### PR DESCRIPTION
**Pull request by kahneraja**

### CODE REVIEW

**Comment by kahneraja** [comment]
fixes #38

File: src/components/MainView.js [hunk 1]
```diff
@@ -75,6 +76,7 @@ export default class MainView extends React.Component {
     this.setState({
       filteredData: data
     });
+    this.refs.dataTable.setWide();
```
Comment (samuelcole) [inline src/components/MainView.js]: propsDidChange() in DataTable should be equivalent?

File: src/components/DataTable.js [hunk 2]
```diff
@@ -11,22 +11,18 @@ export default class DataTable extends React.Component {
...
-    this.setWide();
     window.addEventListener('resize', this.setWide.bind(this));
   }
 
-  componentWillUnmount() {
-    window.removeEventListener('resize', this.setWide);
-  }
-
-  componentDidRecieveProps() {
+  propsDidChange() {
```
Comment (alexyaseen) [inline src/components/DataTable.js]: should this be componentWillReceiveProps()? https://facebook.github.io/react/docs/component-specs.html#lifecycle-methods

Comment (samuelcole) [inline src/components/DataTable.js]: ugh my bad, i was working quickly and guessed the magic method name wrong

**Comment by alexyaseen** [comment]
+1

per @kahneraja and my DM discussion on slack, adding issue #42. We believe we solved it here for this one instance. Removing the event listener wouldn't work otherwise.

**Pull request merged**